tests/harness/executor/BUILD

```diff
@@ -24,17 +23,13 @@ go_library(
     ],
 )
 
-go_test(
+go_binary(
```

Comment:
Question for after merge; how come this change?

Reply:
Since there were no tests the target was more or less a noop. Converting these to actual go tests in the future will be better as I mentioned.